app/javascript/.storybook/storybook.scss

```diff
@@ -3,7 +3,7 @@ body {
   padding: 0;
 
   // This should be Storybook's root element.
-  & > :first-child {
+  #root {
```

Comment:
This was always `id="root"`, even in the older version of Storybook, but in the new version, it's no longer the first child, so be more specific.